bin/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+PATH=$PATH:./node_modules/.bin
+
+mkdir -p build
+rm build/* || true
```

Comment:
`xxxx || true` always ensures that the line exits with a status code of zero. It'll run `xxx` first and if it falls it'll run `true` which always has a status code of zero.

`set -e` makes so if any line has a non-zero status code the script exits immediately